### DISCUSSION TITLE
Does Persistence Work w/Default Missions?

### DISCUSSION THREAD

**Discussion by Rage-GitHub** [Arkensor/EnfusionPersistenceFramework#26] [Q&A]
Just wanted to check, I know that this stores various things by default, but by simply adding this and the database dependency to a server, will it work by default with anything done on then?

**Comment by Arkensor** [comment]
Support for vanilla content and its status is listed in the main readme. Currently, the two game modes are not fully supported, some individual aspects of the AR gameplay overall are.
